requirements.txt

```diff
@@ -217,6 +217,7 @@ SQLAlchemy-Continuum==1.2.4
 SQLAlchemy-Utils==0.32.11
 statsd==3.2.1
 traitlets==4.3.1
+ua-parser==0.7.3
```

Comment:
Have a look at https://zenodo.readthedocs.io/en/latest/development/process.html#python-requirements and the link in that section. First a requirement should go in setup.py (abstract requirement), then a specific version goes in requirements.txt, and requirements.txt list every single package that Zenodo needs (pip freeze) so that we're sure that we don't actually break stuff by suddenly having a third party package upgrade.